app/controllers/MovementsController.scala

```diff
@@ -55,7 +56,7 @@ class MovementsController @Inject()(
 )(implicit ec: ExecutionContext)
     extends BackendController(cc) {
 
-  private val allMessageUnsent: Seq[MovementMessage] => Boolean =
+  private val allMessageUnsent: NonEmptyList[MovementMessage] => Boolean =
```

Comment:
What's this used for?

Reply:
This is the check that determines if a movement is will end up being created as a brand new or as part of a previous movement in which all messages (IE007) are unsubmitted